test/functional/test_framework/authproxy.py

```diff
@@ -101,23 +101,26 @@ def _request(self, method, path, postdata):
         if os.name == 'nt':
             # Windows somehow does not like to re-use connections
             # TODO: Find out why the connection would disconnect occasionally and make it reusable on Windows
+            # Avoid "ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine"
             self._set_conn()
         try:
             self.__conn.request(method, path, postdata, headers)
             return self._get_response()
-        except http.client.BadStatusLine as e:
-            if e.line == "''":  # if connection was closed, try again
+        except (BrokenPipeError, ConnectionResetError):
+            # Python 3.5+ raises BrokenPipeError when the connection was reset
```

Comment:
Is such behavior Windows-specific?

Reply:
Please include the commit you are referring to in the comment. This pull shouldn't change behaviour on non-windows machines in authproxy.